pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -4,11 +4,11 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
please move it back. Ping me for setting up your IDE to avoid this (I know, docs for that should be added but we don't have it yet).

Reply:
I don't understand. Remove it?

Reply:
This is minor. You moved the import to here from line 11. We keep import order consistent to avoid unrelated changes to imports when someone edits a file.

Because you're a comitter I suggested that you configure your IDE to do that:

- In vscode gopls (enabled by default in recent versions) automatically fix import order. 
- In goland it's Ctrl+Alt+S and navigate to Tools | File Watchers, click the Add button and select goimports.